xml/http_servers.xml

```diff
@@ -1724,7 +1724,7 @@
     <param pos="0" name="hw.device" value="VPN"/>
   </fingerprint>
 
-  <fingerprint pattern="^SonicWALL (SSL-?VPN(?: (?:\d+))?) Web Server\.?$">
+  <fingerprint pattern="^SonicWALL SSL-?VPN Web Server$">
```

Comment:
I observed no variants where the capture would have provided value and we don't have any examples.  I removed this part of the regex and will monitor for future banners.